integration/popular_docker_hub_images/popular_docker_images.bats

```diff
@@ -35,347 +37,428 @@ setup() {
+	image="hello-seattle"
+	docker run --rm --runtime=$RUNTIME -i $image | grep "Hello from DockerCon"
 }
 
 @test "[run application] start apachectl in a httpd container" {
-	if docker run --rm --runtime=$RUNTIME -i httpd apachectl -k start | grep "Unable to open logs"; then false; else true; fi
+	image="httpd"
+	if docker run --rm --runtime=$RUNTIME -i $image apachectl -k start | grep "Unable to open logs"; then false; else true; fi
 }
 
 @test "[python application] run python command in a hylang container" {
-	docker run --rm --runtime=$RUNTIME -i hylang bash -c "python -m timeit -s \"L=[]; M=range(1000)\" \"for m in M: L.append(m*2)\""
+	image="hylang"
+	docker run --rm --runtime=$RUNTIME -i $image bash -c "python -m timeit -s \"L=[]; M=range(1000)\" \"for m in M: L.append(m*2)\""
 }
 
 @test "[configuration settings] display config information of a influxdb container" {
-	docker run --rm --runtime=$RUNTIME -i influxdb influxd config
+	image="influxdb"
+	docker run --rm --runtime=$RUNTIME -i $image influxd config
 }
 
 @test "[create directory] start an instance in iojs container" {
-	docker run --rm --runtime=$RUNTIME -i iojs sh -c "mkdir /root/test; ls /root | grep test"
+	image="iojs"
+	docker run --rm --runtime=$RUNTIME -i $image sh -c "mkdir /root/test; ls /root | grep test"
 }
 
 @test "[configuration settings] set nick in an irssi container" {
-	docker run -d --runtime=$RUNTIME irssi irssi -n test
+	image="irssi"
+	docker run -d --runtime=$RUNTIME $image irssi -n test
 }
 
 @test "[java application] display configuration parameters in a jetty container" {
-	docker run --rm --runtime=$RUNTIME -i jetty bash -c "echo 'public class HW{public static void main(String[]a){System.out.println(\"HelloWorld\");}}' > HW.java; ls -l ./HW.java"
+	image="jetty"
+	docker run --rm --runtime=$RUNTIME -i $image bash -c "echo 'public class HW{public static void main(String[]a){System.out.println(\"HelloWorld\");}}' > HW.java; ls -l ./HW.java"
 }
 
 @test "[display version] run jetty container" {
-	docker run --rm --runtime=$RUNTIME -i jetty --version
+	image="jetty"
+	docker run --rm --runtime=$RUNTIME -i $image --version
 }
 
 @test "[ruby application] start jruby container" {
-	docker run --rm --runtime=$RUNTIME -i jruby bash -c "jruby -e \"puts 'Containers'\""
+	image="jruby"
+	docker run --rm --runtime=$RUNTIME -i $image bash -c "jruby -e \"puts 'Containers'\""
 }
 
 @test "[julia application] display message in a julia container" {
-	docker run --rm --runtime=$RUNTIME -i julia bash -c "julia -e 'println(\"this is a test\")'"
+	image="julia"
+	docker run --rm --runtime=$RUNTIME -i $image bash -c "julia -e 'println(\"this is a test\")'"
 }
 
 @test "[display configuration] run kapacitor container" {
-	docker run --rm --runtime=$RUNTIME -i kapacitor bash -c "kapacitord config > kapacitor.conf | ls -l kapacitor.conf"
-}
-
-@test "[display version] display information kibana container" {
-	docker run --rm --runtime=$RUNTIME -i $kibana_image kibana --version
```

Comment:
I see on the [kibana - docker hub page](https://hub.docker.com/_/kibana) that the latest tag is not supported and I remember that this was the reason we added a version of this image to https://github.com/kata-containers/tests/blob/master/versions.yaml#L28-L31

I would prefer to update the version that we track and continue testing this image.